Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts

```diff
@@ -118,7 +120,8 @@ function commitRelease(apiServer: string, apiVersion: string, appSlug: string, u
     tl.debug(`---- url: ${commitReleaseUrl}`);
     let headers = {
         "X-API-Token": token,
-        "User-Agent": userAgent
+        "User-Agent": userAgent,
+        "internal-request-source": "VSTS"
```

Comment:
In addition to the **User-Agent** header that identifies VSTS, Mobile Center also uses the **internal-request-source** header to identify the source.  They had already defined this header and asked us to provide a value.